app/controllers/questions_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
   def show
     @question = Question.find_by(id: params[:id])
+    @solutions = Solution.all
```

Comment:
現状では他の質問の回答も表示されてしまいます。

```suggestion
    @solutions = @question.solutions
```